portfolio/src/main/webapp/scriptMaps.js

```diff
@@ -0,0 +1,177 @@
+class Place {
+    constructor(markerIn, contentIn) {
+        this.marker = markerIn;
+        this.content = contentIn;
+    }
+}
+
+var chelseaContent;
+var hagueContent;
+var cMarker;
+var hMarker;
```

Comment:
Another Google style thing: you use `let` or `const` instead of `var`. (The reason is because the scoping for `var` is very weird.)

Reply:
Also, can these be put inside initMap()?